File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/spacevil/Core/ScrollValue.java
package com.spvessel.spacevil.Core;

public final class ScrollValue {

    public double dX = 0;
    public double dY = 0;

    public void setValues(double dx, double dy) {
        dX = dx;
        dY = dy;
    }

    @Override
    public String toString() {
        return dX + " " + dY;
    }
}

File: SpaceVIL/Core/ScrollValue.cs
using System;

namespace SpaceVIL.Core
{
    public sealed class ScrollValue
    {
        public double DX = 0;
        public double DY = 0;

        public void SetValues(double dx, double dy) {
            DX = dx;
            DY = dy;
        }

        public override String ToString() {
            return DX + " " + DY;
        }
    }
}